app/mailers/receipt_mailer.rb

```diff
@@ -7,4 +7,13 @@ def receipt_email(thesis, user)
          to: @user.email,
          subject: 'Your thesis information submission')
   end
+
+  def transfer_receipt_email(transfer, user)
+    return unless ENV.fetch('DISABLE_ALL_EMAIL', 'true') == 'false' # allows PR builds to disable emails
```

Comment:
I know this logic works and is the same in the other mailer so I'm not suggesting we change it... but also it hurts my brain.